pkg/apis/core/validation/validation.go

```diff
@@ -5212,6 +5193,13 @@ func ValidatePodUpdate(newPod, oldPod *core.Pod, opts PodValidationOptions) fiel
 	var newInitContainers []core.Container
 	for ix, container := range mungedPodSpec.InitContainers {
 		container.Image = oldPod.Spec.InitContainers[ix].Image // +k8s:verify-mutation:reason=clone
+		if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) && utilfeature.DefaultFeatureGate.Enabled(features.SidecarContainers) {
```

Comment:
FYI, I'm adding a resize subresource in #128266. Due to this, I'm moving some of these validation checks to a different function.

Reply:
Thanks for letting me know about it.